Installer.cls

```diff
@@ -18,14 +18,11 @@ XData PM [ XMLNamespace = INSTALLER ]
 <Default Name="REGISTRY" Value="https://pm.community.intersystems.com" />
 <Default Name="TRACKINGID" Value="UA-126752118-1" />
 
-<Namespace Name="%ALL" Code="%DEFAULTDB" Data="%DEFAULTDB" Create="yes" Ensemble="0">
+<Namespace Name="${CURNS}" Code="${CURNSRoutineDB}" Data="${CURNSGlobalDB}" Create="no" Ensemble="0">
```

Comment:
Package Manager should be available from any Namespace, there is no reasons at all to have multiple versions of zpm in different namespaces

Reply:
Looks like, community opinion means nothing
why would you need our review then?

Reply:
There is no way, that it will help current zpm users, would like to use this project
no migration, too many changes for them, they would need to delete all their packages installed to install it again

Reply:
We plan to add some code to the IPM installer to make migration easier (migrating all the storage globals as needed). The goal is to have a smooth transition from ZPM to IPM with no additional work required from consumers of the package manager.

Reply:
When it will happen?
History with IRIS says, it requires more than a year, to get some migration tool

may we keep at least globals the same,
at this time, users with installed packages from ZPM, after upgrade will loose control over installed packages, and will require to reinstall everything